modules/nf-core/variancepartition/dream/templates/dream.R

```diff
@@ -147,13 +153,29 @@ fitmm <- eBayes(fitmm, proportion = opt\$proportion,
 head(fitmm\$design, 3)
 print(colnames(fitmm\$design))
 
-# Write coefficient name from contrast variable and target level (e.g., "treatmentmCherry")
-coef_name <- paste0(opt\$contrast_variable, opt\$contrast_target)
+# If contrast_string is provided, use that for makeContrast
+if (!is.null(opt\$contrast_string)) {
```

Comment:
Just to check, if `$comparison` is NOT supplied, does `opt$contrast_string` really end up as a NULL?

Reply:
The idea would be that if `make_contrast_str` is not supplied it is indeed null, because that's taken care of in the main workflow: 
https://github.com/nf-core/differentialabundance/blob/80c3ba1ddce8d7f843d3d4b6e8aeaa718f416135/workflows/differentialabundance.nf#L380
In case it is not supplied it will use the classic approach.

Reply:
OK, let's assume that works, it's been a while since I've tinkered with this args stuff